src/platform/packages/shared/kbn-scout/src/playwright/page_objects/painless_lab_page.ts

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the "Elastic License
+ * 2.0", the "GNU Affero General Public License v3.0 only", and the "Server Side
+ * Public License v 1"; you may not use this file except in compliance with, at
+ * your election, the "Elastic License 2.0", the "GNU Affero General Public
+ * License v3.0 only", or the "Server Side Public License, v 1".
+ */
+
+import { ScoutPage } from '..';
+
+
+export class PainlessLab {
+  public outputValueElement: Locator;
+  public requestFlyoutHeader: Locator;
+
+  constructor(private readonly page: ScoutPage) {
+    this.outputValueElement = this.page.testSubj.locator('painlessTabs');
+    this.requestFlyoutHeader = this.page.testSubj.locator('painlessLabRequestFlyoutHeader');
+  }
+
+  async goto() {
+    return this.page.gotoApp('dev_tools', { hash: 'painless_lab' });
+  }
+
+  async waitForRenderComplete() {
+    // wait for page to be rendered
+    await this.page.testSubj.locator('painless_lab').waitFor({ state: 'visible' });
+  }
+
+  async setCodeEditorValue(value: string, nthIndex?: number) {
+    await this.page.evaluate(
+      ([editorIndex, codeEditorValue]) => {
+        // The monaco property is guaranteed to exist as it's value is provided in @kbn/monaco for this specific purpose, see {@link src/platform/packages/shared/kbn-monaco/src/register_globals.ts}
+        const editor = window.MonacoEnvironment!.monaco!.editor;
```

Comment:
I see this line is failing the type check locally and in the CI run:

```
proc [tsc] src/platform/packages/shared/kbn-scout/src/playwright/page_objects/painless_lab_page.ts:36:31 - error TS2339: Property 'MonacoEnvironment' does not exist on type 'Window & typeof globalThis'.
--
  | proc [tsc]
  | proc [tsc] 36         const editor = window.MonacoEnvironment!.monaco!.editor;
  | proc [tsc]                                  ~~~~~~~~~~~~~~~~~
```

Would the solution be to import ` '@kbn/monaco'` at the top of `painless_lab_page.ts`?

```ts
import '@kbn/monaco';
```

I ask this because I see the `@kbn/monaco` package exports 

https://github.com/elastic/kibana/blob/8728a2328217408e0a377f037486abbc459acd2d/src/platform/packages/shared/kbn-monaco/src/register_globals.ts#L42-L53

As soon as we import `'@kbn/monaco'` the error seems to go away. And we can simply change it to

```suggestion
        const editor = window.MonacoEnvironment!.monaco.editor;
```

Not sure if this is the recommended way forward, happy to see what the team thinks.

Reply:
This issue solved when moved the pageObject under the plugin.